src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -15,13 +16,19 @@
 using Microsoft.TeamFoundation.Policy.WebApi;
 using Microsoft.TeamFoundation.SourceControl.WebApi;
 using Microsoft.VisualStudio.Services.Common;
-
+using Microsoft.VisualStudio.Services.ReleaseManagement.WebApi;
+using Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Clients;
+using Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Contracts;
+using Microsoft.VisualStudio.Services.WebApi;
 using Newtonsoft.Json;
 
 namespace Roslyn.Insertion
 {
     static partial class RoslynInsertionTool
     {
+        // Currently this is the only release we trigger. We can easily move this over to options when we need this configurable.
+        private const string ReleaseDefinitionName = "Roslyn";
```

Comment:
So we will also need the equivalent for LUT and Project System. So we'll need this configurable right away.

Reply:
The same release definition works for LUT and project system too. So we should be good. Will add comments appropriately. Any suggestions for a name change?

Reply:
How about `ManagedLanguages`?

Reply:
@tmat Do you already have a standard one used for all repo-tools based repos?